backend/src/main/java/de/bund/digitalservice/ris/norms/config/JobrunrExceptionFilter.java

```diff
@@ -0,0 +1,42 @@
+package de.bund.digitalservice.ris.norms.config;
+
+import de.bund.digitalservice.ris.norms.application.port.output.LoadVerkuendungImportProcessPort;
+import de.bund.digitalservice.ris.norms.application.port.output.SaveVerkuendungImportProcessPort;
+import de.bund.digitalservice.ris.norms.domain.entity.VerkuendungImportProcess;
+import de.bund.digitalservice.ris.norms.domain.entity.VerkuendungImportProcessDetail;
+import java.util.List;
+import lombok.extern.slf4j.Slf4j;
+import org.jobrunr.jobs.Job;
+import org.jobrunr.jobs.filters.JobServerFilter;
+import org.springframework.stereotype.Component;
+
+/**
+ * Provides a filter for Jobrunner that makes sure that after retrying the maximum number of times
+ * we write a new status to our error details of the job
+ */
+@Component
+@Slf4j
+public class JobrunrExceptionFilter implements JobServerFilter {
+
+  SaveVerkuendungImportProcessPort saveVerkuendungImportProcessPort;
+  LoadVerkuendungImportProcessPort loadVerkuendungImportProcessPort;
+
+  @Override
+  public void onFailedAfterRetries(Job job) {
+    saveVerkuendungImportProcessPort.saveOrUpdateVerkuendungImportProcess(
+      new SaveVerkuendungImportProcessPort.Command(
+        job.getId(),
+        VerkuendungImportProcess.Status.ERROR,
+        List.of(
+          new VerkuendungImportProcessDetail(
+            "/errors/jobRunFailed",
+            "Tried to import a Normendokumentationspacket the max amount of times but failed",
+            "none"
```

Comment:
this details part is probably going to be a json string for the other cases, so just keeping it empty would probably lead to fewer errors

Reply:
But shouldn't we add the actual error that was encountered? Not sure if through the Job object we can get it, because otherwise we wouldn't know what the real reason was right? And when pulling the process status the info would not be helpful for e-V

Reply:
This case will only happen when the job is running into an unexpected error (and not if something is invalid). In that case we do not want to expose the actual error as we do not know what data it might contain.
The jobrunr job needs to finish successfully even when the uploaded file is invalid to not rerun it multiple times. So an invalid upload is still a successful jobrunr job that updates the VerkuendungsImportProcess status to error (and therefore also handles the writing of the details itself).